src/Test/operations/operate.test.js

```diff
@@ -0,0 +1,31 @@
+import operate from '../../Logic/operate';
+
+describe('all check the operations', () => {
+  describe('1) check the sum', () => {
+    test('should return 3 + 17 = 20', () => {
+      const sum = operate(3, 17, '+');
+      expect(sum).toBe('20');
+    });
+  });
+
+  describe('2) check the minus', () => {
+    test('should return 30 - 10 = 20', () => {
+      const minus = operate(30, 10, '-');
+      expect(minus).toBe('20');
+    });
+  });
+
+  describe('3) check the multiplication', () => {
+    test('should return 30  x 10 = 300', () => {
+      const minus = operate(30, 10, 'x');
+      expect(minus).toBe('300');
+    });
+  });
+
+  describe('3) check the multiplication', () => {
+    test('should return 30  ÷ 10 = 300', () => {
+      const minus = operate(30, 10, '÷');
+      expect(minus).toBe('3');
+    });
+  });
+});
```

Comment:
- [ ]  [ Optional ] I can see you forgot to change the title for the division test as it was a copy of the multiplication from line 18. Kindly edit that test title as well as the test in line 26. I'm sure you wanted to write this => `'should return 30  ÷ 10 = 3'` but you wrote this => `'should return 30  ÷ 10 = 300'`. Please edit that.

- [ ]  [ Required ] Finally, you skipped, or should I forget to write a modulus test. For this project, you must write tests for the functionalities & scenarios of the operate.js. Please add one more test for the modulus (%) operation. 👍